### PR TITLE
Add .NET Bangkok Meetup in the friend-websites

### PR DESCRIPTION
**Pull request by kidchenko**
In this PR we add .NET Bangkok Meetup https://www.meetup.com/dotnet-bkk/ on the list of friend websites.

image

The main changes are on: `friends-websites.yml`

### CODE REVIEW

File: content/data/friends-websites.yml [hunk 1]
```diff
@@ -48,3 +48,7 @@
 - name: Thang Chung - GitHub Repository
   url: https://github.com/thangchung
   about: Codespace by cloud-native developer and architect guy from Vietnam
+
+- name: .NET Bangkok Meetup
+  url: https://www.meetup.com/dotnet-bkk
+  about: Montly meetup about .NET in Bangkok.
```
Comment (aaronamm) [inline content/data/friends-websites.yml]: @kidchenko Mannnnnn proud of you.
However, a little typo `Montly => Monthly`.
After you have fixed this, I will merge your PR.

Thanks.

Comment (kidchenko) [reply content/data/friends-websites.yml]: Thanks @aaronamm fixing today

Comment (kidchenko) [reply content/data/friends-websites.yml]: cc @aaronamm done

**Review by kaiwanyawit-chawankul — APPROVED** [review]
It looks good for me

**Review by aaronamm — APPROVED** [review]

**Pull request merged**